docs/_pages/docs/03-understand/05-trade-escrow.md

```diff
@@ -2,10 +2,60 @@
+* **Taker**: Browse the order book and find an order to your liking. Simply select the "Take Order" option and lock your fidelity bond. Only after the buyer submits their payout invoice will you be required to submit the trade escrow. Use the shown QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats as collateral (escrow). You can now chat with your robot peer to coordinate the fiat payment. *Note: Escrow funds are released to the buyer once you select "Confirm Fiat Received" which settles the order. Only confirm after the fiat has arrived in your account.*
+
+The escrow is not required by the seller until the buyer has locked their payout invoice. As soon as the buyer submits the invoice, the seller then has three hours (by default) to lock the escrow. If the timer runs out, then the order expires and the seller forfeits their bond.
+
+By default, the expiry timer for the escrow is three hours; however, you can customize the order to have an "Escrow Deposit Time-Out" ranging from one hour to eight hours. In other words, modify the time allowed for the seller to submit the escrow funds. Maybe you want a quicker transaction and will set the timer to a maximum of one hour instead of three hours.
+
+Once the fiat payment method is coordinated with your fellow robot, the seller selects "Confirm Fiat Received" to end the trade which releases the escrow funds to the buyer. Only confirm the fiat is received *after* it appears in your possession.
+
+## **Losing Your Trade Escrow**
+
+There are basically three conditions that causes a user to lose their locked trade escrow:
+* Cheat or deceive your peer (and lose the order dispute)
+* Unilaterally cancel the order without your peer’s collaboration (after they sent the fiat payment)
+* Fail to confirm the fiat was received within the given time
+
+The conditions above are expanded upon in additional detail below.
+
+Trying to cheat by intentionally not confirming you received the fiat results in a dispute being automatically opened. You will risk losing that dispute and consequently lose your escrow funds and bond. Refer to [Understand > Bonds](https://learn.robosats.com/docs/bonds/) for additional information on fidelity bonds.
+
+After the trade escrow is locked, the order cannot be cancelled except if both the maker and taker agree to cancel collaboratively. If your peer has confirmed the fiat was sent and you unilaterally cancel the order, then you will risk losing the escrow funds and bond.
+
+Don't forget about your order! If your peer sent the fiat and the order timer expires before you confirm the fiat was received, then you will risk losing your escrow funds and bond since a dispute was opened automatically. Take care to remember your order and back up your robot’s unique token!
```

Comment:
The wording around "escrows" should be something like "releasing" not "losing". Escrows are not lost, they simply are transferred to the new owner. In this case, the escrow goes to the buyer, as he is the rightful owner after sending the fiat (if this is proven in the dispute). In this case, the seller did not "lose" his escrow, he "sold" it for fiat and it was "released" to the buyer. However, he indeed lost the bond!